hotel_roommatik/models/inherited_hotel_folio.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2018 Jose Luis Algara (Alda hotels) <osotranquilo@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import json
+
+from odoo import api, models
+
+
+class HotelFolio(models.Model):
+
+    _inherit = 'hotel.folio'
+
+    @api.model
+    def rm_get_reservation(self, Code):
+        folio_res = self.env['hotel.folio'].search([('id', '=', Code)])
+        json_response = dict()
+        if any(folio_res):
+            folio_lin = folio_res.room_lines
+            json_response = {
+                'Id': folio_res.id,
+                'Arrival': folio_lin[0]['checkin'],
```

Comment:
Perhaps can be good check that 'folio_in' it's not empty ?¿?

Reply:
:+1: yes, I think the same

Reply:
If there is a folio_res there must be at least one folio_lin
do not?

Waiting for those classes and examples on test for Odoo, I tried it by launching a scrip and if it does not exist, the function returns the expected target.